README.md

```diff
@@ -35,15 +35,26 @@ To use LLM-based evaluator, you need to set `OPENAI_KEY` variables.
 ```bash
 export OPENAI_KEY='...'
 ```
+
+### Install the necessary requirements
+Install the dependencies that you need to run the code:
+```bash
+pip install requirements.txt
+```
+
 ### Inference and Evaluation
 The main script to run the evaluation is `./scripts/script_stepwise_phase123.bat`. Below is the command:
 ```bash
 ./scripts/script_stepwise_phase123.bat {models} {phase} {starting_problem_num} {ending_problem_num}
 ```
-`{phase}` defines the phase to be executed in the evaluation. Below are the values of `phase` in the evaluation:
-- 0: run models to get the initial response for either HumanEvalComm or HumanEval. output: file in log/
-- 1: the initial responses are evaluated by the LLM-based evaluator (for HumanEvalComm only). output: file in log/
-- 2: run models again to get the 2nd response based on LLM-based evaluator output (for HumanEvalComm only). output: file in log/
+`{models}` must point to the model that you want to use to run the evaluation.
+`{phase}` defines the phase to be executed in the evaluation. Below are the values of `phase` in the 
+`{starting_problem_num} {ending_problem_num}` define the indices of the first and last problems that the evaluation must be run on.
+
+evaluation:
+- 0: run models to get the initial response of the given model for either HumanEvalComm or HumanEval. output: file in log/
+- 1: the initial responses of the model from the previous step are evaluated by the LLM-based evaluator (for HumanEvalComm only). output: file in log/
+- 2: run models again to get the 2nd response based on the responses got from LLM-based evaluator output (for HumanEvalComm only). output: file in log/
 - 3: extract code and run test cases and other metrics for each problem. input: file in log/  output: file in log/record/
 - 4: compute more metrics for each problem, such as test pass rate, question quality rate, comm. rate, etc. input: file in ./log/record/ output: file in ./result_data/
 - 5: aggregate and display metrics for all problems. output: files in table/
```

Comment:
Can you add the empty table folder to avoid a previous error?

Reply:
Is there anything else like /table folder that might be missing?

Reply:
I think its only the table folder, I will add it